cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectS2ICluster.java

```diff
@@ -128,6 +131,11 @@ public DeploymentConfig generateDeploymentConfig() {
         return dc;
     }
 
+    public DeploymentConfig generateDeploymentConfig(Map annotations) {
```

Comment:
Why overload like this? Why not just change the existing `generateDeploymentConfig()` method, since I think it has only a single caller anyway, right?